tox.ini

```diff
@@ -27,18 +27,18 @@ commands =
       --cover-erase \
       --cover-tests \
       --cover-branches \
-      --ignore-files=test_appengine\.py
+      --ignore-files=contrib/test_appengine\.py
```

Comment:
Sure. I seem to remember trying to use a full path before and it not working. But it's not clear what I'm actually remembering.